server/app/views/admin/programs/ProgramBlockEditView.java

```diff
@@ -391,7 +397,9 @@ private DivTag renderQuestion(
       long blockDefinitionId,
       QuestionDefinition questionDefinition,
       boolean canRemove,
-      boolean isOptional) {
+      boolean isOptional,
+      int questionIndex,
+      int questionsCount) {
```

Comment:
Not blocking: This is pre-existing, but seeing two booleans followed by two ints makes it seem like it'd be really easy to mix up the argument order in this function :(.

Reply:
Is there a way to address it? Create dedicated value object?

Reply:
Yah, the only way I can think of is a dedicated value object. Probably not worth it here. In a perfect world, I'd love something similar to named arguments or destructuring. Definitely not worth blocking this PR over though :D